packages/api-db-mongodb/src/db/user.ts

```diff
@@ -41,12 +43,23 @@ export class MongoDBUserAdapter implements DBUserAdapter {
   }
 
   async createUser({input, password}: CreateUserArgs): Promise<OptionalUser> {
+    const email = normalizeEmail(input.email)
+    const validInput = new Validator(input, {
```

Comment:
Do we maybe want to use a more powerful and more developer friendly validator library like Joi? 
The library is very basic and is using magic strings and since we probably want to start validating more and more I think we should start out with a better foundation. What are your thoughts on that?

Reply:
I think I suggested Joi on Penina's PR, and I definitely agree on this one ;)

Reply:
Yeah that's why I brought it up. My personal favourite is zod because of its typescript integration

Reply:
> https://github.com/omar-dulaimi/prisma-zod-generator
> 
> Just saw this, maybe this would make sense to use

This one we could do later, when we have migrated to prisma :-)